src/SFA.DAS.AdminService.Web/Validators/Roatp/RoatpGatewayPageViewModelValidator.cs

```diff
@@ -16,19 +17,60 @@ public async Task<ValidationResponse> Validate(RoatpGatewayPageViewModel vm)
                 Errors = new List<ValidationErrorDetail>()
             };
 
-            if (string.IsNullOrWhiteSpace(vm.Value))
+            if (string.IsNullOrWhiteSpace(vm.Status))
             {
                 validationResponse.Errors.Add(new ValidationErrorDetail("OptionPass", "Select the outcome of this check"));
             }
             else
             {
-                if (vm.Value =="Fail" && string.IsNullOrEmpty(vm.OptionFailText))
+                if (vm.Status =="Fail" && string.IsNullOrEmpty(vm.OptionFailText))
```

Comment:
Might be worth setting up an enum for statuses

Reply:
will do